From 8b32e5892f9152956a8ad5099fa9712ffddd69a8 Mon Sep 17 00:00:00 2001
From: Thomas Tracy <contact@tjtracy.com>
Date: Fri, 1 Apr 2022 14:28:58 -0400
Subject: [PATCH 0886/1021] chore: [MICROBA-1780] missing copy edit (#902)

We missed a copy edit from the previous PR made for this ticket. Also
changing the id to something more relevant to the message.
---
 .../certificate-status-alert/CertificateStatusAlert.jsx       | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/course-home/outline-tab/alerts/certificate-status-alert/CertificateStatusAlert.jsx b/src/course-home/outline-tab/alerts/certificate-status-alert/CertificateStatusAlert.jsx
index becf78a..2de03e7 100644
--- a/src/course-home/outline-tab/alerts/certificate-status-alert/CertificateStatusAlert.jsx
+++ b/src/course-home/outline-tab/alerts/certificate-status-alert/CertificateStatusAlert.jsx
@@ -66,8 +66,8 @@ function CertificateStatusAlert({ intl, payload }) {
       alertProps.body = (
         <p>
           <FormattedMessage
-            id="learning.outline.alert.cert.when"
-            defaultMessage="This course ends on {courseEndDateFormatted}. Final grades and certificates are
+            id="learning.outline.alert.cert.earnedNotAvailable"
+            defaultMessage="This course ends on {courseEndDateFormatted}. Final grades and any earned certificates are
             scheduled to be available after {certificateAvailableDate}."
             values={{
               courseEndDateFormatted,
-- 
2.34.1

